fedora/infra.py

```diff
@@ -101,7 +101,8 @@ async def oncall_add(self, evt: MessageEvent, username: str) -> None:
             # Just grab the first mxid in the users list on fas
             mxid = mxids[0]
         try:
-            await self.plugin.database.execute(dbq, fasusername, mxid, user.get("timezone", "UTC"))
+            timezone_to_insert = user.get("timezone", "UTC")
+            await self.plugin.database.execute(dbq, fasusername, mxid, timezone_to_insert)
```

Comment:
It looks like you lost your change when you fixed things up:
```suggestion
            await self.plugin.database.execute(dbq, fasusername, mxid, user.get("timezone", "UTC") or "UTC")
```